src/Modules/Frontend/assets/Frontend/webpack/js/_Components.js

```diff
@@ -40,4 +43,10 @@ export class Components {
       })
     }
   }
+
+  static initTogglePasswordInputType () {
+    $('[data-role="toggle-visibility"]').each((index, element) => {
```

Comment:
I find this to generic.
The inner working is very specific, as in: change the html in the element AND also change the type of an input field. While `toggle-visibility` suggests show or hide something. I think the name is not very descriptive. I suggest: `toggle-password-visibility`

Reply:
Fixed a0c40abde